packages/app/src/server/service/g2g-transfer.ts

```diff
@@ -148,27 +210,32 @@ export class G2GTransferPusherService implements Pusher {
     this.crowi = crowi;
   }
 
+  public generateAxiosBaseConfig(tk: TransferKey, config: AxiosRequestConfig = {}): AxiosRequestConfig {
```

Comment:
```suggestion
  public generateAxiosConfig(tk: TransferKey, baseConfig: AxiosRequestConfig = {}): AxiosRequestConfig {
```
意図としてはこっちだった…
ここで作り出したものが base になるというより、base を入れて axios.get なり post なりするときのパラメータを作り出す、という使い方をしているので。